lms/djangoapps/courseware/module_render.py

```diff
@@ -388,7 +386,10 @@ def _fulfill_content_milestones(user, course_key, content_key):
             entrance_exam_enabled = getattr(course, 'entrance_exam_enabled', False)
             in_entrance_exam = getattr(content, 'in_entrance_exam', False)
             if entrance_exam_enabled and in_entrance_exam:
-                exam_pct = _calculate_entrance_exam_score(user, course)
+                # We don't have access to the true request object in this context, but we can use a mock
```

Comment:
I've added some additional commentary to this method explaining some of the madness